### ISSUE TITLE
Emulation of random panned player sound.

### ISSUE THREAD

**Issue by nukeykt** [chocolate-doom/chocolate-doom#656]
Doom has random panned player sound(pitch value goes to separation). So, it should be emulated in Chocolate Doom.
See: https://www.doomworld.com/vb/doom-general/85648-doom-strife-with-random-pitched-sound/

**Comment by jmtd** [comment]
Does/should Heretic, too?

**Comment by nukeykt** [comment]
No. Raven fixed sound code for new DMX api. Strife also has this bug.

**Comment by haleyjd** [comment]
Is this the kind of bug that anyone wants or would benefit from?

**Comment by AXDOOMER** [comment]
This should be emulated.
